docs/src/examples/Cloudy_example.md

```diff
@@ -0,0 +1,518 @@
+# information from Γy
+emulator = Emulator(
+    gaussian_process,
+    input_output_pairs,
+    obs_noise_cov = Γy,
+    normalize_inputs = true
+)
+```
+
+And similarly for `rf-scalar`:
+
+```julia
+kernel_structure = SeparableKernel(
+    LowRankFactor(n_params, nugget),
+    OneDimFactor()
+)
+
+srfi = ScalarRandomFeatureInterface(
+    n_features,
+    n_params,
+    kernel_structure = kernel_structure,
+    optimizer_options = optimizer_options,
+)
+
+retained_svd_frac = 1.0
+input_output_pairs = get_training_points(ekiobj,
+                                          length(get_u(ekiobj))-1)
+
+# Use the medians of the outputs as standardizing factors
+norm_factors = get_standardizing_factors(
+    get_outputs(input_output_pairs)
+)
+
+emulator = Emulator(
+    srfi,
+    input_output_pairs;
+    obs_noise_cov = Γy,
+    normalize_inputs = true,
+    standardize_outputs = true,
+    standardize_outputs_factors = vcat(norm_factors...),
+    retained_svd_frac = retained_svd_frac,
+    decorrelate = true # use SVD to decorrelate outputs
+)
+```
+
+#### Train the emulator
+
+The emulator is trained when we combine the machine learning tool and the data
+into the `Emulator` above. We must also optimize the hyperparameters:
+
+```julia
+optimize_hyperparameters!(emulator)
+```
+
+To test the predictive skill of the emulator, we can e.g. compare its prediction
+on the true parameters to the true data. (One could also hold out a subset of
+the input-output pairs from the training and evaluate the emulator's predictions
+on them).
+
+```julia
+# Check how well the emulator predicts on the true parameters
+y_mean, y_var = Emulators.predict(
+    emulator,
+    reshape(θ_true, :, 1);
+    transform_to_real = true
+)
+
+println("Emulator ($(case)) prediction on true parameters: ")
+println(vec(y_mean))
+println("true data: ")
+println(truth_sample) # what was used as truth
+```
+
+The emulator predicts both a mean value and a covariance.
+
+### Sample the posterior distributions of the parameters
+
+The last step is to plug the emulator into an MCMC algorithm, which is then used to produce samples from the posterior distribution of the parameters. Essentially, the emulator acts as a stand-in for the original forward model (which in most cases of interest is computationally expensive to run) during the MCMC sampling process.
+
+We use the mean across all parameter ensembles from the EKI as the initial
+parameters for the MCMC. Before running the actual MCMC chain, we determine a
+good step size:
+
+```julia
+# initial values
+u0 = vec(mean(get_inputs(input_output_pairs), dims = 2))
+println("initial parameters: ", u0)
+
+# First let's run a short chain to determine a good step size
+yt_sample = truth_sample
+mcmc = MCMCWrapper(
+    RWMHSampling(),
+    truth_sample,
+    priors,
+    emulator;
+    init_params = u0
+)
+
+new_step = optimize_stepsize(
+    mcmc;
+    init_stepsize = 0.1,
+    N = 2000,
```

Comment:
maybe note the stepsize is chosen by running chains of length 2000

Reply:
I added a comment explaining this: "Before running the actual MCMC chain, we determine a good step size by running chains of length `N = 2000`."